examples/bri-3/src/bri/ccsm/contracts/Ccsm.sol

```diff
@@ -0,0 +1,25 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.17;
+
+contract Ccsm {
+  mapping(string => string) public anchorHashStore;
+  event AnchorHashSet(string indexed workgroupId, string anchorHash);
+
+  function setAnchorHash(
+    string calldata _workgroupId,
+    string calldata _anchorHash
+  ) external {
+    require(bytes(_workgroupId).length > 0, 'WorkgroupId cannot be empty');
+    require(bytes(_anchorHash).length > 0, 'AnchorHash cannot be empty');
```

Comment:
it is a hash so it has a maximum length. lets add that check

Reply:
Changes made.